libs/backend/src/cast_vote_records/export.test.ts

```diff
@@ -1028,3 +1031,50 @@ test('areOrWereCastVoteRecordsBeingExportedToUsbDrive returns false when no USB
     areOrWereCastVoteRecordsBeingExportedToUsbDrive({ status: 'no_drive' })
   ).toEqual(false);
 });
+
+test('export and subsequent import of that export', async () => {
+  // Export
+  process.env['VX_MACHINE_TYPE'] = 'scan';
+  for (const sheet of [
+    newAcceptedSheet(interpretedHmpb, sheet1Id),
+    newAcceptedSheet(interpretedHmpbWithWriteIn, sheet2Id),
+    newAcceptedSheet(interpretedBmdBallot, sheet3Id),
+    newRejectedSheet(sheet4Id),
+  ]) {
+    expect(
+      await exportCastVoteRecordsToUsbDrive(
+        mockPrecinctScannerStore,
+        mockUsbDrive.usbDrive,
+        [sheet],
+        { scannerType: 'precinct' }
+      )
+    ).toEqual(ok());
+  }
+  expect(
+    await exportCastVoteRecordsToUsbDrive(
+      mockPrecinctScannerStore,
+      mockUsbDrive.usbDrive,
+      [],
+      { scannerType: 'precinct', arePollsClosing: true }
+    )
+  ).toEqual(ok());
+
+  // Import
+  process.env['VX_MACHINE_TYPE'] = 'admin';
+  const exportDirectoryPaths = await getCastVoteRecordExportDirectoryPaths(
+    mockUsbDrive.usbDrive
+  );
+  expect(exportDirectoryPaths).toHaveLength(1);
+  const exportDirectoryPath = assertDefined(exportDirectoryPaths[0]);
+  const readResult = await readCastVoteRecordExport(exportDirectoryPath);
+  expect(readResult.isOk()).toEqual(true);
+  assert(readResult.isOk());
```

Comment:
Just keep the `assert`?
```suggestion
  assert(readResult.isOk());
```

Or alternatively change the next line to this and remove the assert/expect:

```ts
  const { castVoteRecordIterator } = readResult.unsafeUnwrap();
```

Reply:
Ah yeah, I'd been using a combo of `expect` and `assert` because I find the output of `expect` when a check fails to be more user friendly than `assert`. But `expect` doesn't hint the type checker so an `assert` still remains necessary.

On second thought, though, the combo does feel redundant. Switched to `unsafeUnwrap` where I could: https://github.com/votingworks/vxsuite/pull/4091/commits/e73f2bf922941f82635f28ca6b0faa78640653b8